oslo_vmware/tests/test_rw_handles.py

```diff
@@ -314,7 +314,7 @@ def test_init_failure(self):
                           100)
 
     def test_read(self):
-        chunk_size = rw_handles.READ_CHUNKSIZE
+        chunk_size = 65536
```

Comment:
why is this hard coded needed ?

Reply:
We don't always read the same amount of data anymore (`rw_handles.READ_CHUNKSIZE` is gone with https://github.com/sapcc/oslo.vmware/commit/8894e07b53782d07207a81c069c8fa6a50ea03d5). So I set an abitrary number, which does exactly match the value it had before. It's just used internally by the test here.